packages/expo-updates/ios/EXUpdates.podspec

```diff
@@ -69,6 +71,22 @@ Pod::Spec.new do |s|
     }
   end
 
+  if $expo_updates_create_fingerprint != false
```

Comment:
rather than having a new script phase (and new gradle task), what do you think to rename [createManifest.js](https://github.com/expo/expo/blob/main/packages/expo-updates/scripts/createManifest.js) and **create-manifest-ios.sh** to something like createUpdateResources and merge the manifest / fingerprint invocation entry point all together?

having multiple script phases or gradle tasks may increase the difficulty of debugging when there are any errors from xcode or gradle building system.